core/new-gui/src/app/dashboard/user/service/workflow-version/workflow-version.service.ts

```diff
@@ -83,6 +86,8 @@ export class WorkflowVersionService {
     this.undoRedoService.disableWorkFlowModification();
     // reload the read only workflow version on the paper
     this.workflowActionService.reloadWorkflow(workflow);
+    // clear element ID in URL fragment to avoid highlighting non-existing element
+    this.router.navigate([], { relativeTo: this.route });
```

Comment:
verify that we actually need these

Reply:
Verified that they are not needed. Thank you.